2.5/Dockerfile.fedora

```diff
@@ -0,0 +1,84 @@
+FROM registry.fedoraproject.org/f28/s2i-base:latest
```

Comment:
Why not to use f29?

Reply:
> Apparently it's not available.

@pvalena It's available from yesterday.

> Anyway, it occured to me we could install Ruby from Ruby Module instead, as it's portable across releases...

No problem with that.

IMHO, better to use newer base image because it contains newer packages with more fixes, the image has longer support,... And this can't be solved by using modules.

Reply:
Right, let's do that then.
@pkabelka, FYI: https://docs.fedoraproject.org/en-US/modularity/